### ISSUE TITLE
simplified SingleChainPublisher task context ?

### ISSUE THREAD

**Issue by doudou** [rock-control/control-orogen-robot_frames#2]
`ChainPublisher` is basically a loop over a list of chains, but it does not seem that it really has a benefit over having a single-chain publisher component. The latter would be significantly simpler, both internally and as an interface, since it would not need to have a dynamic port anymore.

I would be willing to implement this component, provided that you're not against it ... @maltewi ?

**Comment by maltewi** [comment]
Would be fine for me. To clarify, you'd propose to implement an additional component named `SingleChainPublisher`? @doudou

**Comment by doudou** [comment]
Exactly. I'll create a PR. Since I'm using SDF, I've implemented it to support both URDF and SDF.
